lib/Fakturoid/Client.php

```diff
@@ -228,6 +228,13 @@ public function createMessage($id, $data)
         return $this->post("/invoices/$id/message.json", $data);
     }
 
+    /* Number lines */
+
+    public function getNumberLines($headers = null)
```

Comment:
Pak poprosím ještě o přejmenování:
```suggestion
    /* Number formats */

    public function getInvoiceNumberFormats($headers = null)
```

Reply:
Upravil jsem to rovnou všude ať je to jednotné